Example-Swift/Photo.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Photo.swift
+//  NYTPhotoViewer
+//
+//  Created by Chris Dzombak on 2/2/17.
+//  Copyright © 2017 NYTimes. All rights reserved.
+//
+
+import UIKit
+
+/// A photo may often be represented in a Swift application as a value type.
+struct Photo {
+    // Ths would usually be a URL, but for this demo we load images from the bundle.
+    let name: String
+
+    let summary: String
+    let credit: String
+
+    let identifier: Int
+}
```

Comment:
Done in https://github.com/NYTimes/NYTPhotoViewer/pull/223/commits/141e05bb820ccb27daf361502c2ab4a50751dfda , except for `identifier` since it's distinct from other Photo properties